script/version-utils.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+const { logger, spawnOrFail, prompt, shouldContinuePrompt, quit, fs, path } = require('./cli-utils');
+const currentVersion = require('../package.json').version;
+
+const isPreRelease = (version) => {
+  return version.split('.')[3] >0;
```

Comment:
Shouldn't this be `> 3` for v3 pre release? Can you please add a comment on how this line of code determines whether a version is preRelease or not?

Reply:
My bad. Clean up code and actually do it wrong. It just check whether there is a build version at the end. Put a comment to explain it.